src/index.js

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable import/no-named-default */
```

Comment:
I have a problem with this one because when I try to ask for separate functions to fix the no-named-default, it says that I can't call 2 imports for the same file. Do you have a solution for that?? If yes, It would be much appreciated.

Reply:
If you exported it as default you don't need to import it as default also. You can import it with whatever name you want for example:
```
import checkboxesEvent, { list } from './_checkboxFunctions.js';
```

Reply:
Oh, okay, I did it. Thank you so much!!